node/src/remote/index.ts

```diff
@@ -236,7 +236,18 @@ export class RemoteTable<T = number[]> implements Table<T> {
   }
 
   async createIndex (indexParams: VectorIndexParams): Promise<any> {
-    throw new Error('Not implemented')
+    const data = {
```

Comment:
There are a lot of parameters in `indexParams` that we are ignoring here.  We might want to check and raise if they are not set to `undefined`.  Otherwise users might be confused when their parameters have no effect.

Reply:
done! also added a test for it unsupported params in the node integration test pr